db/migrate/20230120212413_make_latitude_and_longitude_non_nullable_for_sensors.rb

```diff
@@ -0,0 +1,6 @@
+class MakeLatitudeAndLongitudeNonNullableForSensors < ActiveRecord::Migration[6.0]
+  def change
+    change_column_null :sensors, :latitude, false
+    change_column_null :sensors, :longitude, false
```

Comment:
Since we don't have any sensors without position in the DB we don't need to care about migrating them.